src/registrar/utility/email.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for sending emails."""
+
+import boto3
+
+from django.conf import settings
+from django.template.loader import get_template
+
+
+def send_templated_email(template_name: str, to_address: str, context={}):
+    """Send an email built from a template to one email address.
+
+    template_name is relative to the same template context as Django's HTML
+    templates. context gives additional information that the template may use.
+    """
+
+    template = get_template(template_name)
+    email_body = template.render(context=context)
+
+    ses_client = boto3.client(
+        "sesv2",
+        region_name=settings.AWS_REGION,
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
+    ses_client.send_email(
+        FromEmailAddress=settings.DEFAULT_FROM_EMAIL,
+        Destination={"ToAddresses": [to_address]},
+        Content={
+            "Simple": {
+                "Subject": {"Data": "Thank you for applying for a .gov domain"},
+                "Body": {"Text": {"Data": email_body}},
+            },
+        },
+    )
```

Comment:
If SES is unreachable when the applicant submits their domain application, they are going to get a generic 500 page. 🟥  😢 

The error handling strategy depends on the message*, I think, but the details of why a message failed to send are unlikely to be of interest to the caller, so I recommend logging the real error and then propagating a more generic error upward.

The boto3 library has a pretty useful [page on error handling](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html).

\* For example, in the case of domain application submitted confirmation, it probably doesn't matter at all. In the case of notifying the authorizing official, maybe the system should leave a note for the analysts on that domain request about the failure to send. In the case of a soon-to-expire notice, maybe the domain gets flagged for attention. Etc...

Reply:
I'll catch the error in the `_send_confirmation_email` method and log that it happened and then ignore it there because it seems like the right choice for this particular situation.